source/modulo_core/modulo_core/translators/message_writers.py

```diff
@@ -14,6 +14,46 @@
 StateT = TypeVar('StateT')
 
 
+def get_clproto_msg_type(state: StateT) -> clproto.MessageType:
```

Comment:
Happy do discuss the location and name of this function

Reply:
I would personally be ok with the function staying in the writers for now. I could also see it in a separate file for semantics, but I can't easily think of a case other than this where we need the clproto type  (certainly not something that would justify having a .py just for one function)

Worst case scenario we move it in the future. The `message_writers` would have to include it anyway, so even if we move it later on I would imagine that we don't break something.